frame/grandpa/src/lib.rs

```diff
@@ -580,42 +604,6 @@ impl<T: Trait> Module<T> {
 	}
 }
 
-impl<T: Trait> Module<T> {
```

Comment:
This was unused and the same functionality is implemented on the client where it is needed.